rest_sensors.cpp

```diff
@@ -993,6 +997,119 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                 return REQ_READY_SEND;
             }
 
+            //special part for tuya siren
+            if (sensor->manufacturer().endsWith(QLatin1String("0yu2xgi")))
+            {
+                if (rid.suffix == RConfigMelody)
+                {
+                    int16_t melody = map[pi.key()].toUInt(&ok);
+
+                    QByteArray data;
+                    data.append(static_cast<qint8>(melody & 0xff));
+
+                    if (sendTuyaRequest(task, TaskTuyaRequest , DP_TYPE_ENUM, DP_IDENTIFIER_MELODY, data))
+                    {
+                        updated = true;
+                    }
+
```

Comment:
... empty white space before braces close, also in further lines, please clean this up.